src/hooks/useStarknetkitConnectModal.ts

```diff
@@ -26,7 +26,8 @@ const useStarknetkitConnectModal = (
   const starknetkitConnectModal = async (): Promise<ModalResult> => {
     return await connect({
       ...options,
-      resultType: options?.resultType ?? "connector",
+      skipEmit: true,
+      resultType: undefined,
```

Comment:
I think we can keep `resultType: options?.resultType` (if not passed, it's undefined)
or it would break something?

Reply:
if user passes `connector` it would break Argent connector 😞